BFS/haeun/2667.js

```diff
@@ -0,0 +1,63 @@
+const input = () => {
+    const fs = require('fs');
+    const filePath = process.platform === 'linux' ? '/dev/stdin' : './input.txt';
+
+    const [, ...input] = fs
+        .readFileSync(filePath, 'utf-8')
+        .toString()
+        .trim()
+        .split('\n')
+        .map((line) => line.trim().split('').map(Number));
+    return input;
```

Comment:
```suggestion
    const [length, ...graph] = fs
        .readFileSync(filePath, 'utf-8')
        .toString()
        .trim()
        .split('\n')
        .map((line) => line.trim().split('').map(Number));
    return { N:length[0], graph };
```

이런식으로 주어진 N값을 이용하는건 어떨까요?!